.github/workflows/test.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ["8.2", "8.3", "8.4"]
+        php-versions: ["8.2", "8.3"]
```

Comment:
This can be easily worked around by removing Psalm and PHPCS from the dev-dependencies in `composer.json`. Then in the quality-step you can add them to the `tools` key with a PHP-version of 8.3.

The only downside of this, is that you can no longer run these tools from CLI.

Reply:
Cool, wasn't aware of this... But I basically run it before I commit anything, so I'd like to leave it for now. I believe they'll bump support for it soon...